qt/aqt/profiles.py

```diff
@@ -544,6 +545,18 @@ def new_import_export(self) -> bool:
     def set_new_import_export(self, enabled: bool) -> None:
         self.meta["new_import_export"] = enabled
 
+    def load_last_loaded_profile(self) -> bool:
+        return self.meta.get("load_last_loaded_profile", False)
+
+    def set_load_last_loaded_profile(self, val: bool) -> None:
+        self.meta["load_last_loaded_profile"] = val
+
+    def last_loaded_profile_name(self) -> Optional[str]:
```

Comment:
Modern syntax is `str | None`, which saves you having to import Optional.

Reply:
Thanks, fixed!